websites/S/Steam/presence.ts

```diff
@@ -23,314 +19,471 @@ presence.on("UpdateData", async () => {
 
 		if (document.location.pathname === "/" || !document.location.pathname) {
 			if (subsection) {
-				presenceData.state = `Browsing ${subsection}.`;
-
+				presenceData.state = `Browsing ${subsection}`;
 				presenceData.startTimestamp = browsingTimestamp;
 			} else {
-				presenceData.state = "Home";
-
+				presenceData.state = "Homepage";
 				presenceData.startTimestamp = browsingTimestamp;
 			}
+		} else if (document.location.pathname.includes("/filedetails/")) {
+			presenceData.details = "Viewing a community post";
+			presenceData.state = document
+				.querySelector<HTMLTitleElement>("head > title")
+				.textContent.replace("Steam Community :: ", " ")
+				.replace("::", "|");
+			presenceData.startTimestamp = browsingTimestamp;
```

Comment:
Don't need to start the timestamp every time in each if statement. Simply just add it to the end before the `presence.setActivity(presenceData)` line.

Reply:
Since I was following the way that the way of original creator, I didn't change things much. Just tried to add bunch more...
Also, it actually makes sense to have the timer on some of them (It's not in all of them).